src/extended_audio.c

```diff
@@ -25,9 +26,9 @@
 #endif // EMULATOR
 
 #if HAS_EXTENDED_AUDIO
-u16 voice_map_entries;
-struct VoiceMap voice_map[MAX_VOICEMAP_ENTRIES];
-u16 audio_queue[AUDIO_QUEUE_LENGTH];
+s16 voice_map_entries = -1;
```

Comment:
Can we move this to Transimiter structure? It is shared across a couple of files.

Reply:
yes. also renamed to voice_ini_entries and added enum to avoid magic numbers